modules/authlinkedin/lib/Auth/Source/LinkedIn.php

```diff
@@ -125,4 +120,23 @@ public function finalStep(&$state) {
 
 		$state['Attributes'] = $attributes;
 	}
+
+    /**
+     * @param $array
+     * @param string $prefix
+     *
+     * @return array
+     */
+    protected function flatten($array, $prefix = '') {
+        $result = [];
+        foreach($array as $key => $value) {
```

Comment:
According to the PSR-2 standard that we are following, there must be an space between the `foreach` keyword and the subsequent opening parenthesis.